docs/install.rst

```diff
@@ -382,20 +382,21 @@ of this section.
 
 On the other hand, if you wish to call Sphinx manually to build the
 documentation, you will need to make sure that a number of dependencies are
-installed. The easiest way to install these is to specify ``[docs]`` when
+installed.  If you use conda, the easiest way to install the dependencies is
+with::
+
+    conda install -c astropy sphinx-astropy
+
+Without conda, you install the dependencies by specifying ``[docs]`` when
 installing ``astropy`` with pip::
 
-    pip install -e .[docs]
+    pip install -e '.[docs]'
```

Comment:
Quotes are not needed.
```suggestion
    pip install -e .[docs]
```

Reply:
Yes, quotes are need in zsh, which is now default on mac.

Reply:
```
(astropy) ➜  astropy git:(enhan-desc-#9660) pip install -e .[docs]
zsh: no matches found: .[docs]
```

Reply:
Some people know how I feel about Macs. I am just gonna refrain myself.

Reply:
zsh is not a mac thing ;) and at least now they have a modern shell.

Reply:
what about defining which shell we're assuming? I don't care whether it's bash or zsh or xonsh. (I assume some of the current bash snippets would not work anyway on tcsh which is still the default on the ancient linux distros, etc.)

Reply:
OT, but zsh has changed my life with oh-my-zsh and having color-coded git branch + dirty tag in the prompt, + branch name autocomplete etc etc.  I know one can get this in other ways, but I never bothered until zsh just made it trivial.

Reply:
oh, but I have all of those with bash, too :) (xonsh is the next level though, I saw a few really nice stuff folks do with it, I just never bothered to switch)